doc/adr/0014-marconi-query-interface.rst

```diff
@@ -0,0 +1,39 @@
+ADR 14: Marconi Query Interface
+======================================================
+
+Date: 2022-10-27
+
+Author(s)
+---------
+
+Kayvan Kazeminejad <kayvan.kazeminejad@iohk.io>
+
+Status
+------
+
+Draft
+
+Context
+-------
+
+Some DApps need to index and query a portion of the blockchain for a specific list of addresses. `Marconi <https://plutus-apps--763.org.readthedocs.build/en/763/adr/0004-marconi-initiative.html>`_ provides a general solution for indexing the blockchain data.  The query interface adds reporting capabilities on top of Marconi for a specific list of addresses for both Haskell and non-Haskell applications.
+
+Decision
+--------
+
+- We will build the interface on top of the Marconi indexer with minimal impact on the Marconi infrastructure
+
+- The interface indexes for the addresses that the user/DApps provides
+
+- The query interface may be used both as an executable or a Haskell library
+
+- The query interface supports `JSON-RPC 2.0 <https://www.jsonrpc.org/>`_ on top of HTTP
```

Comment:
Active voice.

"We will use `JSON-RPC 2.0 <https://www.jsonrpc.org/>`_ on top of HTTP as the communication protocol for the query interface."